04/app.js

```diff
@@ -6,17 +6,17 @@ const bodyEl = document.querySelector('body');
 const addClassToElement = function (e) {
 
     if (e.toElement.className === 'child') {
+        e.stopPropagation();
         divsList.forEach(function (divEl) {
             setTimeout(function () { divEl.classList.add('clicked'); }, divEl.dataset.time);
         });
     } else {
         this.classList.add('clicked');
+        e.stopPropagation();
```

Comment:
Skoro dodałeś w obu miejscach to można by było to zrobić na samym początku.
Trochę jak w matematyce "wyciągnać przed nawias"